storage/ndb/src/kernel/blocks/dbtup/AggInterpreter.cpp

```diff
@@ -0,0 +1,2138 @@
+                fprintf(stderr, "%x ", *(dec_buf_ptr + i));
+              }
+              fprintf(stderr, "\n");
+              if (dec_ret == E_DEC_OVERFLOW) {
+                return ZAGG_DECIMAL_CONV_OVERFLOW;
+              } else {
+                return ZAGG_DECIMAL_CONV_ERROR;
+              }
+            }
+            // if (frag_id_ == 0) {
+            //   if (scale != 0) {
+            //     fprintf(stderr, "Moz-Intp: Load NDB_TYPE_DECIMALUNSIGNED[double] %lf\n",
+            //         registers_[reg_index].value.val_double);
+            //   } else {
+            //     fprintf(stderr, "Moz-Intp: Load NDB_TYPE_DECIMALUNSIGNED[uin64] %lu\n",
+            //         registers_[reg_index].value.val_uint64);
+            //   }
+            // }
+          break;
+
+          default:
+            // assert(0);
+            return ZAGG_LOAD_COL_WRONG_TYPE;
+        }
+        break;
+      case kOpLoadConst:
+        type = (value & 0x03E00000) >> 21;
+        reg_index = (value & 0x000F0000) >> 16;
+        assert(type == NDB_TYPE_BIGINT || type == NDB_TYPE_BIGUNSIGNED ||
+               type == NDB_TYPE_DOUBLE);
+        ResetRegister(&registers_[reg_index]);
+        registers_[reg_index].type = AlignedType(type);
+        registers_[reg_index].is_unsigned = IsUnsigned(type);
+        registers_[reg_index].is_null = false;
+        switch (type) {
+          case NDB_TYPE_BIGINT:
+            registers_[reg_index].value.val_int64 =
+                sint8korr(reinterpret_cast<char*>(&prog_[exec_pos]));
+            // fprintf(stderr, "Moz-Intp: LoadConst[%u] NDB_TYPE_BIGINT %ld\n",
+            //     reg_index, registers_[reg_index].value.val_int64);
+            break;
+          case NDB_TYPE_BIGUNSIGNED:
+            registers_[reg_index].value.val_uint64 =
+                uint8korr(reinterpret_cast<char*>(&prog_[exec_pos]));
+            // fprintf(stderr, "Moz-Intp: LoadConst[%u] "
+            //                 "NDB_TYPE_BIGUNSIGNED %lu\n",
+            //     reg_index, registers_[reg_index].value.val_uint64);
+            break;
+          case NDB_TYPE_DOUBLE:
+            registers_[reg_index].value.val_double =
+                doubleget(reinterpret_cast<unsigned char*>(
+                      &prog_[exec_pos]));
+            // fprintf(stderr, "Moz-Intp: LoadConst[%u] NDB_TYPE_DOUBLE %lf\n",
+            //     reg_index, registers_[reg_index].value.val_double);
+            break;
+          default:
+            // assert(0);
+            return ZAGG_LOAD_CONST_WRONG_TYPE;
+        }
+        exec_pos += 2;
+        break;
+      case kOpMov:
+        reg_index = (value & 0x0000F000) >> 12;
+        reg_index2 = (value & 0x00000F00) >> 8;
+
+        registers_[reg_index] = registers_[reg_index2];
+        break;
+      case kOpSum:
+        reg_index = (value & 0x000F0000) >> 16;
+        agg_index = (value & 0x0000FFFF);
+
+        ret = Sum(registers_[reg_index], &agg_res_ptr[agg_index], print_);
```

Comment:
Turn print_ into a #define that maps to false in production and print_ in debug mode